tasks/task-arguments.unit.test.js

```diff
@@ -0,0 +1,57 @@
+describe('Task arguments', () => {
+  let args
+
+  beforeEach(() => {
+    jest.resetModules()
+
+    args = {
```

Comment:
I feel like there's a jest reason but how come this can't be included in the declaration on line 2?

Reply:
It's a good point, and would probably work fine, but because objects are ["pass by reference"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#comparing_objects) in JavaScript there's a risk that one test will modify `args` affecting other tests

So I'm "resetting" `args` as a test fixture on every **beforeEach()**

Reply:
Makes sense to me! Resolving.